### DISCUSSION TITLE
Offline Mode

### DISCUSSION THREAD

**Discussion by ghost** [robertodoering/harpy#510] [Ideas]
I was about to go a journey without an Internet and thought it would be cool to have some new tweets offline to check later.

For example, fetching all followed account within 24 hours including comments + media.
Fetching specified accounts
Fletching a list
Fetching a topic, etc.

An option where we could specify what we want to fetch, how long, and contain what (comments), etc. would be cool.

What you think?

**Comment by robertodoering** [comment]
Caching and pre-fetching Tweets is definitely nice and will come to harpy eventually. Also when we add system notifications we will have to fetch Tweets in the background as well so it will be nice to have a Tweet cache by then.
